crates/sui-json-rpc-types/src/sui_transaction.rs

```diff
@@ -371,6 +371,9 @@ pub struct SuiTransactionEffectsV1 {
     /// The epoch when this transaction was executed.
     pub executed_epoch: EpochId,
     pub gas_used: SuiGasCostSummary,
+    /// The version that every modified (mutated or deleted) object had before it was modified by
+    /// this transaction.
+    pub modified_at_versions: Vec<(ObjectID, SequenceNumber)>,
```

Comment:
clarifying question for me: so for this `SuiTransactionEffect`, `modified_at_versions`[i] tells me the `ObjectID` that was previously modified (before this current one) and the `SequenceNumber` at which a modification occurred previously

Reply:
yes, the input object info basically